benchmark/benchmark.php

```diff
@@ -89,6 +91,14 @@ function runWordpress(bool $jit): array {
     return runValgrindPhpCgiCommand('wordpress', [$dir . '/index.php'], cwd: $dir, jit: $jit, warmup: 50, repeat: 50);
 }
 
+function runLaravelDemo(bool $jit): array {
+
+    $dir = __DIR__ . '/repos/laravel-demo-10.10';
+    cloneRepo($dir, 'https://github.com/nielsdos/laravel-demo-app.git');
```

Comment:
If there are no problems with the initial Laravel benchmark, we can also obtain the code from https://github.com/laravel/laravel.git .

In that case we need to run the following command:
````
./artisan key:generate
````
Ref. https://github.com/laravel/laravel/blob/v10.0.0/composer.json#L46-L48

If we need to fix the version, it may be a good idea to specify a clone tag.

Reply:
We need to fix the version to avoid influences from code changes within Laravel. Even when fixing the composer version we need to fix dependency versions too, even in the event the dependency may disappear. Therefore the easy solution, like we do with other benchmarks, is to simply put everything including the vendor dir in a separate repo.

Reply:
Understood! I certainly thought that was desirable. thank you!